### PR TITLE
Update translations + Add metadata

### LINKED ISSUE #57: Add appdata to be translated
Translating `appdata.xml` file will help people to know about your app in the AppCenter. For example mostly French users won't download the app if it's description is in English, 'cause they will think that the app isn't translated and won't take time to read if its not translated.

### PR DESCRIPTION
**Pull request by NathanBnm**
Hi there! I updated translations and added metadata translations (This will fix #57)  :smile: 

I also updated French translations.

**This PR is ready for review**

### CODE REVIEW

**Comment by NathanBnm** [comment]
@BharatKalluri #59 should make this PR not failing

File: po/ru.po [hunk 1]
```diff
@@ -10,6 +15,70 @@ msgstr ""
...
+msgstr ""
+
+#: src/Widgets/BasicOptions.vala:21
+msgid "Start Time :"
+msgstr ""
+
+#: src/Widgets/BasicOptions.vala:25
+msgid "End Time :"
+msgstr ""
+=======
```
Comment (ryonakano) [inline po/ru.po]:
```suggestion
```

I guess this is the biggest reason the CI still fails, maybe it appeared while resolving conflicts.

**Review by ryonakano — CHANGES_REQUESTED** [review]

File: po/ru.po (hunk 1 above)
Comment (NathanBnm) [reply po/ru.po]: Whoops! You're right I didn't see that.

**Comment by NathanBnm** [comment]
@BharatKalluri the PR should pass this time.

**Comment by ryonakano** [comment]
Seems like the build still fails and something wrong with the translation files.

**Review by BharatKalluri — APPROVED** [review]

**Comment by ryonakano** [comment]
@NathanBnm Is the removal of `po/com.github.bharatkalluri.gifup.pot` intentional?

**Comment by NathanBnm** [comment]
It was not intentional. I'll do a PR to fix this.

**Pull request merged**